Documentation/security/network/encryption-ipsec.rst

```diff
@@ -171,6 +171,29 @@ commands:
        15:16:21.627699 IP 10.60.1.1 > 10.60.0.1: ESP(spi=0x00000001,seq=0x57e4), length 100
        15:16:21.628408 IP 10.60.1.1 > 10.60.0.1: ESP(spi=0x00000001,seq=0x57e5), length 100
 
+.. note::
+
+    When monitoring network traffic on a node, it is normal to observe both the encrypted
+    (node-to-node ``ESP``) and decrypted packets (pod-to-pod). This occurs as,
+    once a packet is decrypted, it is recirculated back to the same interface for further processing.
+    Depending on the ``tcpdump`` filter applied, it is possible to capture different types of packets,
+    but this does not indicate that encryption is not functioning correctly:
+    
+    1. Both the two packets: no filter or ``"src <node-ip> or src <pod-ip>""``.
```

Comment:
```suggestion
    1. Both the two packets: no filter or ``"src <node-ip> or src <pod-ip>"``.
```

Besides the typo, I have a question for my understanding: filtering for `"src <node-ip> or src <pod-ip>"` to see both the encrypted and unencrypted packet would only work with tunneling while targeting the tunnel device (e.g. cilium_vxlan) with `tcpdump`, correct?

Reply:
I did some investigation locally, playing with the following parameters to see changes during the traffic capture.

```yaml
bpf:
  masquerade: false
autoDirectNodeRoutes: true
ipv4NativeRoutingCIDR: 10.244.0.0/16
ipv6NativeRoutingCIDR: fd00:10:244::/56
endpointRoutes:
  enabled: true
enableIPv4Masquerade: false
enableIPv6Masquerade: false
enableMasqueradeRouteSource: false
```

The difference is that instead of observing traffic in `cilium_vxlan` you can see it directly in the node `eth0` with the routes (`ip r`) inserted accordingly. Routing CIDR correspond to the Kind `podSubnet` param and the two pod can ping. However, I wasn't able to avoid the translation between podIP `10.244.2.189` to its relative nodeIP `10.244.2.92` while sending ICMP Echo Requests.

I'd be glad to hear suggestions, I'm sure there's something I'm missing :disappointed: 

IMHO, to coherently fill this section of the doc, I would avoid mentioning the filter ``"src <node-ip> or src <pod-ip>"`` that can cause confusion due to the many possible setup. Instead I would propose `no filter or ESP plus specific filter for the the protocol used by the pods (such as ``"ecp or icmp"`` for ping)`. Let me know :smiley:

Reply:
>IMHO, to coherently fill this section of the doc, I would avoid mentioning the filter "src <node-ip> or src <pod-ip>" 
SGTM!